examples/exotica_examples/src/planner.cpp

```diff
@@ -65,12 +62,17 @@ void run()
         ROS_INFO_STREAM_THROTTLE(0.5, "Publishing states to rviz ...");
         while (ros::ok())
         {
-            any_problem->getScene()->Update(solution.row(t).transpose());
-            any_problem->getScene()->getSolver().publishFrames();
-
+            int i = 1;
+            if(t==0 || t==solution.rows()-1) i = 30;
```

Comment:
Why is ``i=30`` - maybe comment that this is arbitrary and should be cleaned up/addressed later